ext/RMagick/rmpixel.c

```diff
@@ -633,7 +639,10 @@ Pixel_from_MagickPixel(const MagickPixel *pp)
     pixel->blue    = ROUND_TO_QUANTUM(pp->blue);
     pixel->opacity = ROUND_TO_QUANTUM(pp->opacity);
 
-    return Data_Wrap_Struct(Class_Pixel, NULL, destroy_Pixel, pixel);
+    obj = Data_Wrap_Struct(Class_Pixel, NULL, destroy_Pixel, pixel);
```

Comment:
I am wondering if Pixel should be changed to use a `QuantumPixelPacket` instead?

Reply:
Yeah, I’m think so

Reply:
I tested this fix. Getting the black value is possible but setting the black value isn't. the `Magick::Pixel#black=` method is gone.

Reply:
With this way, I think it is difficult to enable `Magick::Pixel#black=` ...